src/wp-includes/block-template.php

```diff
@@ -17,6 +17,40 @@ function _add_template_loader_filters() {
 	}
 }
 
+/**
+ * Renders a warning screen for empty block templates.
+ *
+ * @since x.x
+ *
+ * @param WP_Block_Template $block_template The block template object.
+ * @return string           The warning screen HTML.
```

Comment:
The coding standards don't require white space alignment for the return annotation.

```suggestion
 * @return string The warning screen HTML.
```